bot/exts/evergreen/cheatsheet.py

```diff
@@ -92,7 +93,7 @@ async def cheat_sheet(self, ctx: Context, *search_terms: str) -> None:
             search_string = quote_plus(" ".join(search_terms))
 
             async with self.bot.http_session.get(
-                    URL.format(search=search_string)
+                    URL.format(search=search_string), headers=HEADERS
```

Comment:
You could just directly pass in the HEADERS here instead of defining a variable.

Reply:
Thanks for the review, I could do that but i would like it as a separate constant being passed it.

Reply:
Sure! It was just a suggestion, it looks fine like this too.